imagersite/imager_profile/apps.py

```diff
@@ -0,0 +1,11 @@
+from __future__ import unicode_literals
+
+from django.apps import AppConfig
+
+
+class ImagerProfileConfig(AppConfig):
+    name = 'imager_profile'
+
+    def ready(self):
+        """Have handler listen."""
+        import handler
```

Comment:
This causes tests to fail - at least, in Py3.5. You should use `from imager_profile import handler` instead.